app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,102 @@
+class AlbumsController < ApplicationController
+  before_action :set_album, only: [:show, :edit, :update, :destroy]
+  # before_action :authenticate_user!, except: [:show, :index]
+  # GET /albums
+  # GET /albums.json
+  def index
+    @albums = Albums.all
+  end
+
+  # GET /albums/1
+  # GET /albums/1.json
+  def show
+    @album = Albums.find(params[:id]) 
```

Comment:
there was a problem with the show, and the solution was to pluralize the Album also